build_tools/pkgci/external_test_suite/sdxl_scheduled_unet_gpu_rocm_gfx942.json

```diff
@@ -0,0 +1,41 @@
+{
+  "config_name": "gpu_rocm",
+  "iree_compile_flags" : [
+    "--iree-hal-target-backends=rocm",
+    "--iree-rocm-target-chip=gfx942",
+    "--iree-opt-const-eval=false",
+    "--iree-codegen-transform-dialect-library=${IREE_TEST_PATH_EXTENSION}/attention_and_matmul_spec.mlir",
+    "--iree-global-opt-propagate-transposes=true",
+    "--iree-global-opt-enable-fuse-horizontal-contractions=true",
+    "--iree-flow-enable-aggressive-fusion=true",
+    "--iree-opt-aggressively-propagate-transposes=true",
+    "--iree-opt-outer-dim-concat=true",
+    "--iree-vm-target-truncate-unsupported-floats",
+    "--iree-llvmgpu-enable-prefetch=true",
+    "--iree-opt-data-tiling=false",
+    "--iree-codegen-gpu-native-math-precision=true",
+    "--iree-codegen-llvmgpu-use-vector-distribution",
+    "--iree-rocm-waves-per-eu=2",
+    "--iree-execution-model=async-external",
+    "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline, util.func(iree-preprocessing-pad-to-intrinsics))",
+    "--iree-scheduling-dump-statistics-format=json",
+    "--iree-scheduling-dump-statistics-file=compilation_info.json"
+  ],
+  "iree_run_module_flags": [
+    "--device=hip",
+    "--parameters=model=real_weights.irpa",
+    "--module=sdxl_scheduled_unet_pipeline_fp16_rocm.vmfb",
```

Comment:
Would we need a gfx942 version of https://github.com/nod-ai/SHARK-TestSuite/blob/3603a453b3777fac9af4506a3dc0b3d87587fd47/iree_tests/pytorch/models/sdxl-scheduled-unet-3-tank/sdxl_scheduled_unet_pipeline_fp16_rocm.vmfb ?

https://github.com/iree-org/iree/actions/runs/9864966241/job/27241318002?pr=17842#step:12:49

```
=================================== FAILURES ===================================
___ IREE compile and run: sdxl-scheduled-unet-3-tank::gpu_rocm_real_weights ____
Error invoking iree-run-module
Error code: 1
Stderr diagnostics:
iree/runtime/src/iree/hal/drivers/hip/native_executable.c:186: INTERNAL; HIP driver error 'hipErrorNoBinaryForGpu' (209): no kernel image is available for execution on the device; mismatched target chip? missing/wrong bitcode directory?; while invoking native function hal.executable.create; while calling import; 
```

If so, we may want to leave the SDXL tests off in this initial PR and roll forward with the "special models" changes... or revert https://github.com/nod-ai/SHARK-TestSuite/pull/271 to bring back those files and get us unstuck 🤔

Reply:
Putting down as an expected run failure for now. With the special model test suite changes in flight, should be back to running as intended for numerics as well pretty soon